show-line-of-sight-between-geoelements/src/main/java/com/esri/arcgismaps/sample/showlineofsightbetweengeoelements/screens/MainScreen.kt

```diff
@@ -0,0 +1,103 @@
+/* Copyright 2024 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.esri.arcgismaps.sample.showlineofsightbetweengeoelements.screens
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Slider
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.viewmodel.compose.viewModel
+import com.arcgismaps.toolkit.geoviewcompose.SceneView
+import com.esri.arcgismaps.sample.sampleslib.components.SampleTopAppBar
+import com.esri.arcgismaps.sample.showlineofsightbetweengeoelements.components.SceneViewModel
+
+/**
+ * Main screen layout for the sample app
+ */
+@Composable
+fun MainScreen(sampleName: String) {
+
+    // Define the viewmodel of this sample
+    val sceneViewModel: SceneViewModel = viewModel()
+
+    // Retrieve any changes to the z value from SceneViewModel
+    val currentZValue = sceneViewModel.currentZValue.collectAsState().value
+
+    // Defined in order to keep the z value in the positive range
+    val offset = 100
+
+    Scaffold(topBar = { SampleTopAppBar(title = sampleName) },
+        content = {
+            Column {
+                Box(
+                    modifier = Modifier
+                        .fillMaxSize()
+                        .weight(1f)
+                ) {
```

Comment:
- This `Box` is the only child of the parent `Column` which would fill that max size. I think it is effectively the same as not having the Column in the first place. 
- This being said, we typically shouldn't use a Box to overlay Composables on top of the MapView/SceneView. This could block the bottom attribution bar or the top license for developer usage. Some situations like using a FAB are fine but probably not a full-width `Row`. 
- I think we should use the `paddingValues` provided by the Scaffold in the parent `Column`'s modifier here since it will ensure all the children of this parent layout will use the padding. It helps adjust the layout to fit in a topAppBar or the bottom navigational gestures.